test/test_rospy/test/unit/test_rospy_core.py

```diff
@@ -116,6 +116,8 @@ def test_loggers(self):
             base, ext = os.path.splitext(this_file)
             if ext == '.pyc':
                 this_file = base + '.py'
+            if os.path.sep == '\\':
+                this_file = this_file.replace(os.path.sep, r'\\')
```

Comment:
Wouldn't it make more sense to `re.escape(this_file)`?

Same below.

Reply:
thanks for sharing this with us! will change to use `re.escape`